humans.txt

```diff
@@ -0,0 +1,11 @@
+/* TEAM */
+  William Durand
```

Comment:
Can you please remove those two spaces at the beginning of each line from this section?